core/templates/dev/head/pages/exploration_editor/post_publish_modal_directive.html

```diff
@@ -47,6 +47,11 @@ <h3>Awesome!</h3>
     <p>
       Now that you're a published Oppia teacher, you can share your creation.
     </p>
+    <div>
+      Here's your exploration link
+     <br>
+     <b><[explorationLink]></b>
```

Comment:
Why is the link not a html element?

Reply:
The link is meant to be copied and shared to other people , so I guess redirecting the creator back to the exploration isn't necessary .

Reply:
@Aashish683 take a look at the "embed exploration modal", which you can access by going to the player view of an exploration, clicking the info icon at the top next to the title, and clicking on the "Embed" button in the bottom right. I mention this for two reasons:

(i) That provides a more copy-able user interface, which you might want to mimic here.
(ii) That modal also shows a link to the exploration for others to share/embed, and you might want to see how they do it.

Reply:
Okay @seanlip , I will look into this .
Thanks :)